ocr.py

```diff
@@ -6,6 +6,7 @@
 from PIL import Image
 
 pp = pprint.PrettyPrinter(indent=4)
+HEADERS = [u'backlog', u'doing', u'done']
```

Comment:
Looks good for now, but we'll need to revisit and discuss "header keywords". A list of other possible keywords:

* WIP
* Work in progress
* Finished
* Completed